site/docs/sample-queries/aws_route53.md

```diff
@@ -5,35 +5,39 @@ slug: '/aws_route53'
 
 # AWS Route53
 
-Install the AWS Route53 module for hosted zones. Read more about AWS Route53 hosted zones [here](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-working-with.html).
+Install the AWS Route53 module for hosted zones. Read more about AWS Route53 hosted
+zones [here](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-working-with.html).
 
 ```sql
-SELECT * FROM iasql_install('aws_route53');
+SELECT *
+FROM iasql_install('aws_route53');
 ```
 
 ## Create a hosted zone
 
-Create a hosted zone and new record within it. Finally, `apply` it.
+Create a hosted zone and new record within it, and apply to the cloud.
 
 ```sql TheButton
+SELECT iasql_begin();
 INSERT INTO hosted_zone (domain_name)
 VALUES ('iasqlsample.com');
 
 INSERT INTO resource_record_set (name, record_type, record, ttl, parent_hosted_zone_id)
 SELECT 'test.iasqlsample.com', 'CNAME', 'example.com.', 300, id
 FROM hosted_zone
 WHERE domain_name = 'iasqlsample.com';
-
-SELECT * FROM iasql_apply();
+SELECT iasql_commit();
```

Comment:
Same.

Reply:
This has a `SELECT * FROM` afterwards. I think we also should keep using transactions here.